admiraldev.Rproj

```diff
@@ -1,5 +1,4 @@
 Version: 1.0
-ProjectId: cdbce8e6-0934-430c-a76c-0c8cffbb9203
```

Comment:
FYI this is a new "feature" they've added to Rstudio IDE. If you update your install, it will be auto-added to each project

Reply:
ah i saw this happen in admiral as well.  

is the Project ID unique to each user?

Reply:
> is the Project ID unique to each user?

I haven't investigated much, but I think it's just a shift from using the folder name is the ID to this unique ID for each project.